### DISCUSSION TITLE
Configuration/Settings Page Location/Access

### DISCUSSION THREAD

**Discussion by noidesserts** [flameshot-org/flameshot#3707] [Q&A]
I just downloaded Flameshot (Windows 11), and while the PrntScrn shortcut works, I can't find the config page.

**Comment by mmahmoudian** [comment]
I don't have Windows to test this quickly, but if you use Start menu and run Flameshot, you will see the Flameshot icon in the tray area. Right-clicking on that would give you a menu from which you can open Configurations.

**Comment by GaborElAyoubi** [comment]
@noidesserts you can right click on the tray icon in the taskbar:
![image](https://github.com/user-attachments/assets/86aa0122-9284-436d-92ac-b39bb66e328b)

Hope this helps
